ratpack-session/src/main/java/ratpack/session/internal/AllowAllSessionTypeFilter.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ratpack.session.internal;
+
+import ratpack.session.SessionTypeFilter;
+
+public class AllowAllSessionTypeFilter implements SessionTypeFilter {
```

Comment:
This should have a name like "Unsafe" or "Insecure" in the name and also be marked Deprecated.

Reply:
done